### DISCUSSION TITLE
Item tag for stackable soups

### DISCUSSION THREAD

**Discussion by MarioSMB** [vectorwing/FarmersDelight#430] [Ideas]
There is an option in the configuration file to list which soups can be stacked, it would be nice if this could be extended to an item tag so that mods and datapack creators can add support for this feature. Thank you.

**Comment by vectorwing** [comment]
We faced a few issues and crashes while attempting this change. The soup stack increase is a mixin, and using tags to determine which items are affected was problematic (some mods access these before tags are bound).

For now, it kinda needs to be done through the config. :( But I'll take a look again later.

**Reply by MarioSMB** [reply]
Ah that's a shame, thanks for looking into it! I can see how this would pose a challenge as I'm not aware of any mods that provide such functionality.

Here's hoping a solution presents itself, ironically it's probably those same mods that would benefit from this the most!
